tests/test_moxunit_runtests.m

```diff
@@ -0,0 +1,228 @@
+function test_suite=test_moxunit_runtests
+    initTestSuite;
+
+function test_moxunit_runtests_basics
```

Comment:
Seems a bit large to me for a single unit test. Would it make sense for this to be broken down into multiple test cases?